### ISSUE TITLE
[ECS] OnVehicleDeath problem

### ISSUE THREAD

**Issue by A1dyyyy** [ikkentim/SampSharp#352]
```
[Event]
public void OnVehicleDeath (Vehicle vehicle, Player killerid)
{
Console.WriteLine ($ "Destroy {vehicle.Entity.Handle}");
}
```
It does not work.

**Comment by fvandillen** [comment]
You really should be more specific. What doesn't work? Do you get an error, exception?

**Comment by A1dyyyy** [comment]
@fvandillen No, I am not getting an exception error, it just doesn't work

```
public class ServerVehicles: ISystem
{
	[Event]
	public void OnVehicleDeath (Vehicle vehicle, Player killerid)
	{
                Console.WriteLine("Work."); // nothing happens
		Console.WriteLine ($ "Destroy {vehicle.Entity.Handle}"); // nothing happens
	}
}
```

**Comment by ikkentim** [comment]
closed in 0b6b2e5
